src/streamlink_cli/argparser.py

```diff
@@ -708,6 +708,22 @@ def hours_minutes_seconds(value):
 
     Default is 10.0.
     """)
+transport.add_argument(
+    "--hls-segment-ignore-names",
+    metavar="NAMES",
+    type=comma_list,
+    help="""
+    A comma-delimited list of segment names that will not be fetched.
+
+    Example: --hls-segment-ignore-names 000,001,002
+
+    This will ignore every segment that ends with 000.ts, 001.ts and 002.ts
+
+    Default is None.
+
+    Note: The --hls-timeout must be increased, to a time that is longer than the ignored break.
```

Comment:
I guess this can't be automated, right? Segment lengths are dependent on the streaming service (may even be variable) and there could also be multiple segments in a row which get removed.

Reply:
i could change `timeout=self.timeout` to a bigger value
if this command is used

but that's not really automated